gem/spec/fixtures/earlgrey_yaml/earlgrey_gem.yml

```diff
@@ -0,0 +1,4 @@
+TeacherUITests:
```

Comment:
Change to something general like - SampleEarlGreySwiftTests ?

Reply:
Done